docker/dbsmigration/run.sh

```diff
@@ -42,19 +42,18 @@ fi
 
 # use service configuration files from /etc/secrets if they are present
 cdir=/data/srv/current/config/$srv
-# check if /etc/secrets area contain at least one dbs configuration files
 cfiles="DBSDef.py DBSMigrate.py DBSGlobalReader.py DBSGlobalWriter.py DBSPhys03Reader.py DBSPhys03Writer.py Migration.py"
+pushd $cdir
```

Comment:
Alan, this PR should not change logic of another deployment, and the changes your provide are not correct. They will disallow the use case when we do not have these files in config area, i.e. when we want to run service with default (image provided) config files. Please revert the changes

Reply:
That's a good point. I didn't consider the use case where we want to run the service with the default image configuration. 
I can revert these changes, but note that your implementation will only work for a *single* configuration file provided in services_config. If someone wants to provide 2 or more files, and remove whatever comes in the image, it will not work.